task.ps1

```
@@ -1,12 +1,13 @@
-$location = "uksouth"
+$location = "switzerlandnorth"
 $resourceGroupName = "mate-azure-task-11"
 $networkSecurityGroupName = "defaultnsg"
 $virtualNetworkName = "vnet"
 $subnetName = "default"
 $vnetAddressPrefix = "10.0.0.0/16"
 $subnetAddressPrefix = "10.0.0.0/24"
 $sshKeyName = "linuxboxsshkey"
-$sshKeyPublicKey = Get-Content "~/.ssh/id_rsa.pub" 
+$publicKeyPath = Join-Path $HOME '.ssh/ssh-key-ubuntu.pub'
```

The script reads the SSH public key from `$HOME/.ssh/ssh-key-ubuntu.pub`. Add a `Test-Path` check and a helpful error message before `Get-Content` so the script fails early with a clear message if the key file is missing.